bin/collator/src/parachain/service.rs

```diff
@@ -1009,14 +1011,15 @@ where
 #[cfg(feature = "evm-tracing")]
 pub async fn start_astar_node(
     configuration: StartupConfiguration,
+    additional_config: AdditionalConfig,
 ) -> sc_service::error::Result<(
     TaskManager,
     Arc<TFullClient<Block, astar::RuntimeApi, NativeElseWasmExecutor<astar::Executor>>>,
 )> {
     start_node_impl::<astar::RuntimeApi, astar::Executor, _, _>(
```

Comment:
What about changing the `fn start_node_impl` too?

Reply:
Sure, we can just propagate additional_config to `start_node_impl`